### ISSUE TITLE
Need UX for Mobile App

### ISSUE THREAD

**Issue by Bjorn248** [codegnv/lens-reactnative#3]
Basic User flow and home screen for mobile app

Find a design person?

**Comment by sirbugsbunny** [comment]
hey, do you need help with UX? if yes please let me know!

**Comment by Bjorn248** [comment]
@sirbugsbunny absolutely! We are always in need of UX folks. See #2 for a rough initial feature set
